src/languages/en.js

```diff
@@ -1,4 +1,169 @@
+/* eslint-disable max-len */
 export default {
     hello: 'Hello',
     phoneCountryCode: '1',
+    welcomeText: [
```

Comment:
This file will get impossible to follow very quick. I think we need to namespace this a bit better. Something like:
```
  common: {
    yes: 'Yes',
    no: 'No',
    ... other things that are common and used in lots of pl
  }
  updateAppModal: {
    updateApp: 'Update App',
    ... other texts in this modal...
  }
  IOUConfirmationList: {
    whoWasThere: 'WHO WAS THERE?',
    ...
  }
...
```
Makes sense?